linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/rule.yml

```diff
@@ -9,10 +9,13 @@ description: |-
     each <tt>ClientAliveCountMax</tt> message. If the SSH server does not
     receive a response from the client, then the connection is considered idle
     and terminated.
-
     To ensure the SSH idle timeout occurs precisely when the
     <tt>ClientAliveInterval</tt> is set, set the <tt>ClientAliveCountMax</tt> to
-    value of <tt>0</tt>.
+    value of <tt>0</tt>. This profile sets <tt>ClientAliveCountMax</tt> to
```

Comment:
This and the previous lines are incorrect, but the OCIL text is correct, so it just needs to be copy-pasted here.

Reply:
Changes made!